crates/sncast/tests/e2e/deploy.rs

```diff
@@ -330,14 +330,11 @@ async fn test_contract_not_declared() {
     ];
 
     let snapbox = runner(&args);
-    let output = snapbox.assert().success();
+    let output = snapbox.assert().failure();
 
     assert_stderr_contains(
         output,
-        indoc! {r"
-        command: deploy
-        error: An error occurred in the called contract[..]Class with hash[..]is not declared[..]
-        "},
+        "Error: Couldn't retrieve contract class with hash: 0x1",
```

Comment:
This is kind of "breaking" in terms of that we are loosing an explanation of why is this contract missing. Previously it said that it is not required. I'm not sure if that's good.

Reply:
Made an issue [#2554](https://github.com/foundry-rs/starknet-foundry/issues/2554#issue-2564353877)